docker-cpp-demos/rtsp-docker/start_rtsp_in_docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Demo GStreamer Sample Application for RTSP Streaming to Kinesis Video Streams
+# To be run inside the Docker container
+
+cd /opt/amazon-kinesis-video-streams-producer-sdk-cpp/build/
+
+# Start the demo rtsp application to send video streams
+export LD_LIBRARY_PATH=/opt/amazon-kinesis-video-streams-producer-sdk-cpp/open-source/local/lib:$LD_LIBRARY_PATH
+
+# $3 for STREAM_NAME and $4 is RTSP_URL
```

Comment:
Is it possible to revise the start up script so that params order are not jumbled?

Reply:
Done.

Reply:
Done